### PR TITLE
Added php 7.2 on travis

### PR DESCRIPTION
**Pull request by samnela**

### CODE REVIEW

**Comment by dunglas** [comment]
It looks like there is some work to do to make CI green.

**Comment by samnela** [comment]
We couldn't use deps=low as long as there is still support for php 5 otherwise the build will fail

**Comment by dunglas** [comment]
You mean with PHP 7.2? What's the error?

File: .travis.yml [hunk 1]
```diff
@@ -44,7 +49,7 @@ install:
     - if [[ $deps = 'beta' ]]; then composer config minimum-stability beta; fi;
     - if [[ $deps != 'low' ]]; then composer update --prefer-dist --no-progress --no-suggest --ansi; fi
     - if [[ $deps = 'low' ]]; then composer update --prefer-dist --no-progress --no-suggest --prefer-stable --prefer-lowest --ansi; fi
-
+    - if [[ $(phpenv version-name) = '7.2' ]] && [[ $deps = 'low' ]]; then composer require phpunit/phpunit:^5.0 --dev --prefer-dist --update-with-dependencies; fi
```
[the remark this answers is not in the document]
Comment (samnela) [reply .travis.yml]: Thanks! As soon as possible I will try this solution

File: composer.json [hunk 2]
```diff
@@ -21,9 +21,9 @@
   },
   "require-dev": {
     "doctrine/doctrine-bundle": "^1.6",
-    "phpunit/phpunit": "^4.0 || ^5.0 || ^6.0",
     "symfony/finder": "^2.8 || ^3.0 || ^4.0",
-    "symfony/framework-bundle": "^2.8 || ^3.0 || ^4.0"
+    "symfony/framework-bundle": "^2.8 || ^3.0 || ^4.0",
+    "symfony/phpunit-bridge": "^3.3 || ^4.0"
```
Comment (samnela) [inline composer.json]: `^3.3` is only version constraint that works well for `deps=low`

Comment (Toflar) [reply composer.json]: I think we can just update to `^3.4` as 3.3 is not maintained anymore anyway. I'll do that on master directly.

**Comment by Toflar** [comment]
Thanks @samnela!

File: composer.json (hunk 2 above)
Comment (Toflar) [reply composer.json]: Ah sorry, I thought this was regarding SF. Misread that, forget my comment then 😄

**Pull request merged**